js/adapt-assessmentArticleModel.js

```diff
@@ -748,6 +749,8 @@ const AssessmentModel = {
 
     if (_isAssessmentComplete) this._checkIsPass();
 
+    this._setAssessmentOwnershipOnChildrenModels();
```

Comment:
Hmm

Reply:
I've made sure to specify which models should be marked as part of the assessment at each stage, rather than having `_setAssessmentOwnershipOnChildrenModels` rely on `_originalChildModelsStore`

Reply:
This is looking great 👍